### PR TITLE
Sherbet: Change active link colour

### PR DESCRIPTION
**Pull request by mikachan**
This updates the active link colour in Sherbet, to increase the colour contrast.

Part of https://github.com/WordPress/twentytwentythree/issues/182.

### CODE REVIEW

File: styles/sherbet.json [hunk 1]
```diff
@@ -136,6 +136,11 @@
 					"link": {
 						"color": {
 							"text": "var(--wp--preset--color--contrast)"
+						},
+						":active": {
+							"color": {
+								"text": "var(--wp--preset--color--contrast)"
+							}
```
Comment (madhusudhand) [inline styles/sherbet.json:154]: works good in post content, but the issue persists in comments -> links

**Review by madhusudhand — COMMENTED** [review]
Can you also consider updating the colors for button states too.

image

File: styles/sherbet.json (hunk 1 above)
Comment (mikachan) [reply styles/sherbet.json:154]: Thanks! I've updated the comment link active states here https://github.com/WordPress/twentytwentythree/pull/234/commits/7d43c3ce8a4ea790a2be8c86d19d05bc0c97a4e3

**Comment by mikachan** [comment]
> Can you also consider updating the colors for button states too.

This has been done on trunk, so I've merged trunk into this branch so we can see all the changes together.

**Comment by mikachan** [comment]
I've pushed some more improvements for the button states so that the active and focus states look like this:

image

I'm going to merge this in so it's included in Beta 3, but please open any follow up issues/PRs if needed!

**Pull request merged**